scripts/scil_compute_sf_from_sh.py

```diff
@@ -43,14 +44,15 @@ def _build_arg_parser():
     p.add_argument("--extract_as_dwi", action="store_true",
```

Comment:
Also, I dont understand why extract_as_dwi is a parameters (it confused me at first).
Why not remove this parameters and simply say in --in_bval that the resulting SF are actually interpretable as DWI.
(It would be important to say that this is a 'related' script to scil_compute_sh_from_signal.py)

Also after that you can keep a warning for --b0 if --in_bval isn't there
Then --out_bvecs and --out_bvals could be both optional if the person want it or not.